includes/Dashboard.php

```diff
@@ -122,10 +122,17 @@ public function enqueue_assets( $hook_suffix ) {
 
 		wp_set_script_translations( self::SCRIPT_HANDLE, 'web-stories' );
 
+		wp_register_style(
```

Comment:
We already have a `load_admin_fonts` in `Story_Post_Type`. Should we try and reuse this method.

Reply:
The only thing you could really reuse is the `Fonts::get_font()` and `wp_enqueue_style()` parts.

But since Google Sans is an **unlisted** font on Google Sans, it's not in `fonts.json` and thus `Fonts::get_font()` doesn't work.